lib/BoundaryClickWatcher/index.js

```diff
@@ -18,7 +19,8 @@ class BoundaryClickWatcher extends Component {
     onMouseEnter() {},
     onMouseLeave() {},
     className: '',
-    alwaysListen: false
+    alwaysListen: false,
+    ignoreClass: null
```

Comment:
should this be an empty string `''` rather than `null` if it's PropType is `string`?

Reply:
Doesn't really matter, it'll just treat null as the prop not existing, and wont throw up any propType errors